RollGen/Utils.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace RollGen
+{
+    public static class Utils
```

Comment:
Regardless of if this is injectable or static, we should add unit tests around this class.